tools/validation_files.py

```diff
@@ -1,53 +1,72 @@
 import sys
-from auxiliaries.pipeline_auxiliaries import load_table_to_dict, sample2bc_schema
+from auxiliaries.pipeline_auxiliaries import load_table_to_dict
 import os
 import re
 import json
 import jsonschema
+import datetime
+
+def is_valid_Json_file(json_path, schema, logger):
```

Comment:
Function is not called anywhere

Reply:
I gonna use it to check the validation of the new JSON files - like the JSON for phase 1. 
That is why I put it in the share file, I thought it belongs there.